lib/workers/global/config/parse/index.ts

```diff
@@ -22,6 +22,27 @@ export async function parseConfigs(
   const cliConfig = cliParser.getConfig(argv);
   const envConfig = envParser.getConfig(env);
 
+  if (cliConfig?.dryRun === 'true') {
+    logger.warn('cli config dryRun property has been changed to full');
+    cliConfig.dryRun = 'full';
+  }
+  if (envConfig?.dryRun === 'true') {
+    logger.warn('env config dryRun property has been changed to full');
+    envConfig.dryRun = 'full';
+  }
+  if (cliConfig?.dryRun === 'false' || cliConfig?.dryRun === 'null') {
+    logger.warn(
+      'cli config dryRun property has been changed to null, running with normal mood.'
```

Comment:
```suggestion
      'cli config dryRun property has been changed to null, running with normal mode.'
```

I think you mean `normal mode` here. 😉

But maybe Renovate bot can really get moody, and be grumpy about it's working hours/conditions. Maybe it needs a break. 🤣

Reply:
🤣🤣🤣